controller/src/schemas/user.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+from controller.src.schemas.base import BaseWithMetadata
+
+
+class User(BaseWithMetadata):
+    _extra_fields = ["policy", "features"]
+    _top_level_fields = ["email", "full_name"]
+
+    email: str
+    full_name: Optional[str] = None
```

Comment:
`Optional` and `= None` is the same thing. Unless for pydantic its not.

Reply:
my bad, will update on all schemas, it is just redundant